### ISSUE TITLE
Adaptive cards in tabs, light color is white

### ISSUE THREAD

**Issue by guillewu** [MicrosoftDocs/msteams-docs#8384]
When I use:
```
{
            "type": "TextBlock",
            "text": "${description}",
            "wrap": true,
            "color": "Light"
        },
```

The text doesn't show up, as it's white (not gray). Is there a way for the msteams team to fix this bug?

Thanks

**Comment by sayali-MSFT** [comment]
@guillewu - Thanks for reporting your issue.
We will check this at our end and will get back to you.

**Comment by guillewu** [comment]
@sayali-MSFT thanks, looking forward to the fix

**Comment by sayali-MSFT** [comment]
@guillewu -Got the reply from engneering teams that "The behaviour is inline with the expectation from adaptivecard.io"
Changing the theme to Microsoft Teams, it seems the behaviour is inline with adaptivecard.io.
![Screenshot 2023-04-24 165727](https://user-images.githubusercontent.com/93527375/233983346-c926c607-a1d3-4146-8c95-fe55d07aaf5b.png)

**Comment by guillewu** [comment]
@sayali-MSFT thank you for your reply. However, I don't understand your answer. Does that mean that in "light" mode, there is no option for a light color that results in a light gray font color?

**Comment by sayali-MSFT** [comment]
@guillewu -Unfortunately, as of now, the same is not possible out of the box. As mentioned above, the light grey behaviour is from Bot Framework Chat theme. In Microsoft Teams theme in the portal itself we can see that white color is used.
